frontend/src/components/pages/CampOverview/PickupDropoffTableRow.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import { Text, Tr, Td, HStack } from "@chakra-ui/react";
+
+import { ReactComponent as SunriseIcon } from "../../../assets/icon_sunrise.svg";
+import { ReactComponent as SunsetIcon } from "../../../assets/icon_sunset.svg";
+
+const weekdayMap: { [key: number]: string } = {
+  1: "Monday",
+  2: "Tuesday",
+  3: "Wednesday",
+  4: "Thursday",
+  5: "Friday",
+};
+
+// findTime consumes an array of dates and a specified weekday (i.e. Sunday = 0, Monday = 1, etc.).
+// Returns the time of the first date in the array that matches the weekday (i.e. 8:00 AM) or XX:XX if none found.
+const findTime = (dates: Date[] | undefined, weekday: number) => {
```

Comment:
confirm with Andy that sessions don't run on weekends (this might change the number of rows we need etc.)

Reply:
See the slack thread for more details: https://uwblueprint.slack.com/archives/C02R9QK34UB/p1664765611920599

Reply:
Just updated the branch to incorporate this. The way I'm doing it is I get a set of unique weekdays amongst both the early dropoff and late pickup date arrays, then I only map the weekdays in those sets. If we ever have an early dropoff time selected for a specific weekday but not a late pickup time selected (or vice versa), the late pickup time will show XX:XX. We will never have a row that says "XX:XX" in both ED and LP columns, since this won't even be mapped. If both the ED or LP objects in the camper DTO are empty, the table won't print and we'll only show some text saying that no dates were specified.

Reply:
this sounds good. We need to support sessions on Sunday and Saturday too not just weekdays

Reply:
Yup I added that, you may be looking at an older version?

Reply:
oh lol, just saw that you mentioned weekdays only. lemme review the code again now